cas-server-documentation/protocol/CAS-Protocol-Specification.md

```diff
@@ -22,7 +22,7 @@ Contributors:
 -   Robert Oschwald [CAS 3.0]
 -   Misagh Moayyed
 
-Version: 3.0.1
+Version: 3.0.2
```

Comment:
That said, I think we may anticipate newer changes to the protocol to specially support multifactor authentication. so that might be a good excuse to bump this to 3.1, but given that I don't know we'd be able to do all of that for 4.2, I'd still keep it at a patch release for now.